client/signup/steps/clone-jetpack/index.jsx

```diff
@@ -0,0 +1,144 @@
+import SignupActions from 'lib/signup/actions';
+import ActivityLogItem from 'my-sites/stats/activity-log-item';
+
+import QuerySites from 'components/data/query-sites';
+import QueryActivityLog from 'components/data/query-activity-log';
+import QueryRewindState from 'components/data/query-rewind-state';
+import QuerySiteSettings from 'components/data/query-site-settings';
+
+import { getSiteGmtOffset, getSiteTimezoneValue, getActivityLogs } from 'state/selectors';
+import {
+	adjustMoment,
+	getActivityLogQuery,
+	getStartMoment,
+} from 'my-sites/stats/activity-log/utils';
+
+class CloneJetpackStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+		signupDependencies: PropTypes.object,
+	};
+
+	selectNew = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'new',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	selectMigrate = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'migrate',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	renderStepContent = () => {
+		const { originSiteName, destinationSiteName, translate } = this.props;
+
+		return (
+			<div>
+				<div className="clone-jetpack__column">
+					<Card className="clone-jetpack__image-card">
+						<img
+							alt="upgrade"
+							className="clone-jetpack__image"
+							src="/calypso/images/upgrades/thank-you.svg"
+						/>
+					</Card>
+					<Card className="clone-jetpack__text-card">
+						<Button className="clone-jetpack__button" onClick={ this.selectNew }>
+							{ translate( 'Create new Jetpack connection' ) }
+						</Button>
+						<p className="clone-jetpack__description">
+							{ translate(
+								'Keep the plan on %(originSiteName)s and create a new Jetpack connection on %(destinationSiteName)s.',
+								{
+									args: { originSiteName, destinationSiteName },
+								}
+							) }
+						</p>
+					</Card>
+				</div>
+				<div className="clone-jetpack__column">
+					<Card className="clone-jetpack__image-card">
+						<img
+							alt="upgrade"
+							className="clone-jetpack__image"
+							src="/calypso/images/upgrades/thank-you.svg"
+						/>
+					</Card>
+					<Card className="clone-jetpack__text-card">
+						<Button className="clone-jetpack__button" onClick={ this.selectMigrate }>
+							{ translate( 'Migrate Jetpack plan' ) }
+						</Button>
+						<p className="clone-jetpack__description">
+							{ translate(
+								'Browse your event history and choose an earlier state to clone from.'
+							) }
+						</p>
+					</Card>
+				</div>
+			</div>
+		);
+	};
+
+	render() {
+		const {
+			flowName,
+			stepName,
+			positionInFlow,
+			signupProgress,
+			originSiteName,
+			translate,
+		} = this.props;
+
+		const headerText = translate( 'Your Jetpack connection' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Jetpack Connection' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind/clone-site)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).